Update documentation to new format and dependency version (#86)
ballerina-platform/module-ballerinax-redis: README.md

# Ballerina Redis Connector

[![Build](https://github.com/ballerina-platform/module-ballerinax-redis/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/actions?query=workflow%3ACI)
[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-redis.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/commits/master)
[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)

> [Redis](https://redis.io/) is an open source, in-memory data structure store that you can use as a database, cache, and message broker. It supports data structures such as strings, hashes, lists, sets and sorted sets.

Reviewer: Suggestion: "[Redis](https://redis.io/) is an open-source, in-memory data structure store that you can use as a database, cache, and a message broker. It supports data structures such as strings, hashes, lists, sets, and sorted sets.

Reviewer 2: We should avoid using the hyphen with regard to the term open source[1].
[1] https://developers.google.com/style/word-list?hl=en#letter-o

The Redis [Ballerina](https://ballerina.io/) connector provides operations to connect and interact with the Redis server over the network.

> For more information, go to the module(s).

Reviewer: Suggestion: "For more information about configurations and operations, click the relevant module in [Ballerina Documentation - ballerinax/redis](https://lib.ballerina.io/ballerinax/redis/latest)."

Suggesting this revision where we link to the connector page instead of module pages because it seems like you can't point to the "latest" doc when you directly link to the module docs. Is there a way?

Also, if you add more modules to a connector, updating this documentation can be easily missed.

Reviewer 3: Thing is, then all docs are pointed to the latest version. Before publish, heard to find the link as well. Futher, this link may change if we move Ballerina central to a different domain. As we want to point this README.md to module.md of `this` version, best thing is to use a relative path to module.md

# Ballerina Redis Connector

[![Build](https://github.com/ballerina-platform/module-ballerinax-redis/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/actions?query=workflow%3ACI)
[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-redis.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/commits/master)
[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)

> [Redis](https://redis.io/) is an open source, in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets.

Reviewer 2: Suggestion:
[Redis](https://redis.io/) is an open-source, in-memory data structure store that you can use as a database, cache, and message broker. It supports data structures such as strings, hashes, lists, sets, and sorted sets.

> This connector provides operations for connecting and interacting with Redis server over the network. 

Reviewer 2: ```suggestion
This connector provides operations to connect and interact with the Redis server over the network. 
```

> For more information about configuration and operations, go to the module. 

Reviewer 2: Suggestion:
For more information on connector configuration and operations, see [`ballerinax/redis`](redis/Module.md).

- [`ballerinax/redis`](redis/Module.md)

# Building from the Source
## Setting Up the Prerequisites

> 1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).

Reviewer 2: Suggestion:
Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).

Author: Will add

Author: As we have a common format across all the connectors. If we change this it will be inconsistent when compared with other connectors

Author: Changed

   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)

   * [OpenJDK](https://adoptopenjdk.net/)

        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.

2. Download and install [Ballerina Swan Lake Beta2](https://ballerina.io/). 

## Building the Source

> Execute the commands below to build from the source.

Reviewer 2: ```suggestion
Execute the following commands to build from the source:
```

> 1. To build the package:

Reviewer 2: Suggestion:
Let's use bullet points instead of numbers because users don't need to perform the steps in a sequence.

Author: Yes, I will change. That is more meaningful

Author: As we have a common format across all the connectors. If we change this it will be inconsistent when compared with other connectors

Author: Changed according with above suggestion

    ```    
    bal build -c ./redis
    ```
2. To run the without tests:
    ```
    bal build -c --skip-tests ./redis
    ```
# Contributing to Ballerina
As an open source project, Ballerina welcomes contributions from the community. 

> For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/main/CONTRIBUTING.md).

Reviewer 2: Suggestion: 
"For more information, see the [Contribution Guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/main/CONTRIBUTING.md)."

# Code of Conduct
All contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).

# Useful Links

> * Discuss about code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).

Reviewer 2: ```suggestion
* Discuss about code changes of the Ballerina project via [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```
